conductr_cli/constants.py

```diff
@@ -17,7 +16,6 @@
                                        '{}/plugins'.format(DEFAULT_CLI_SETTINGS_DIR))
 DEFAULT_SANDBOX_IMAGE_DIR = os.path.abspath(os.getenv('CONDUCTR_SANDBOX_IMAGE_DIR',
                                                       '{}/images'.format(DEFAULT_CLI_SETTINGS_DIR)))
-DEFAULT_SANDBOX_INTERFACE = os.getenv('CONDUCTR_SANDBOX_INTERFACE', host.loopback_device_name())
 DEFAULT_SANDBOX_ADDR_RANGE = os.getenv('CONDUCTR_SANDBOX_ADDR_RANGE', '192.168.1.0/24')
```

Comment:
Can we hold off this discussion after I came back on 11 Jan? I'd definitely like to get some thoughts from @huntc regarding this matter.

The address range `127.0.0.0/24` will include `127.0.0.1` which is reserved for localhost. Using this address range will mean sandbox for ConductR 1 and ConductR 2 will try to obtain `127.0.0.1` on port `9005`, which I think may be problematic especially if ConductR 1 containers doesn't stop properly.

Because of this, I'm somewhat hesitant to use `127.0.0.0/24`.